src/main/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionary.java

```diff
@@ -63,6 +66,11 @@
         if (URI == null) {
             URI = "file:" + REFERENCE_SEQUENCE.getAbsolutePath();
         }
+        if (OUTPUT == null) {
+            // determine the name for the dict file in the same way as CachingIndexedFastaSequenceFile.checkAndCreate
+            final String fastaExt = REFERENCE_SEQUENCE.getAbsolutePath().endsWith("fa") ? "\\.fa$" : "\\.fasta$";
```

Comment:
I used the same code as in `CachingIndexedFastaSequenceFile.checkAndCreate()` to maintain the behaviour of that class. In addition, if I will port this to Picard it does not have (I guess) teh commons-io dependency.

Should I change it anyway?